ui/app/templates/components/oidc/assignment-form.hbs

```diff
@@ -26,80 +26,80 @@
       </h1>
     </p.levelLeft>
   </PageHeader>
-  <form {{on "submit" (perform this.save)}}>
-    <div class="box is-sideless is-fullwidth is-marginless">
-      <FormFieldLabel for="name" @label="Name" />
-      <input
-        autocomplete="off"
-        spellcheck="false"
-        value={{@model.name}}
-        disabled={{not @model.isNew}}
-        class="input field {{if this.modelValidations.name.errors 'has-error-border'}}"
-        {{on "input" this.handleOperation}}
-        data-test-input="name"
+{{/unless}}
```

Comment:
This large diff is just changes in indentation from adding the `{{#unless}}` block and removing the `div`